### DISCUSSION TITLE
[Portal Engine] Can't create Data object data pool document

### DISCUSSION THREAD

**Discussion by Maksonis** [pimcore/pimcore#13262] [Q&A / Support] [answered]
Hi,

when I try to create document of type:

![image](https://user-images.githubusercontent.com/17277953/192797255-65e80e48-f41d-43ff-a1ec-d2a39958f55a.png)

I get this error:

![image](https://user-images.githubusercontent.com/17277953/192797333-1634cca2-9dca-4cc2-881c-197142835a4c.png)

On demo I get this:

![image](https://user-images.githubusercontent.com/17277953/192797379-69449434-d4f9-4849-bdcd-781b2802302a.png)

Is this a bug ?

**Comment by kingjia90** [comment, marked as answer]
Thank you for reporting! 
I've verified and already pulled a request and will be fixed in the 2.4.2 release.
If you want to hotfix in the meanwhile, you just have to add a `?` here  `protected ?string $dataObjectClass;` on line 26 in `portal-engine\src\Model\Configuration\DataPool\DataObjectConfig.php`

**Reply by Navino16** [reply]
Hi,

Any ETA until 2.4.2 release ?

**Reply by fashxp** [reply]
Tomorrow
